README.md

```diff
@@ -1,24 +1,73 @@
-# README
+## Hotel Reservstion 🏩
+This application is a web application for final capstone project that you can find your favorite Hotel around the world and reserve it for a specific date, find information about a hotel at details page and cancel a reservation.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Project Documentation 📄
+- [Live Demo]()
+- [API Documentation](http://127.0.0.1:5000/api-docs/index.html)
 
-Things you may want to cover:
+## Team members: 4 (Dozie Udeagha, SaharSaba Amiri, Mean Mohammad, Bright Kweku Ahiadeke )
 
-* Ruby version
+# Built With 🛠️
+- Programming Languages: HTML, CSS, JavaScript, Ruby.
+- Frameworks: React, Redux, Rails and Bootstrap.
+- GitHub Linters.
+- Code Editor: VS Code.
 
-* System dependencies
+## Kanban Board
+- Click this [link](https://github.com/users/maenkhraisha/projects/6) to view the Kanaban Board
+- Click this [link](https://docs.google.com/document/d/1GcBpwqr2NsF9umMA4qthxuBxnEjcLagS8p_tt5CwHPc/edit?usp=sharing) to view the initial state of the kanban board
+- Click [here](https://github.com/maenkhraisha/hotel-reservation) for the main branch of the repo
+# Getting Started
+To get a local copy up and running follow these simple steps.
 
-* Configuration
+### Prerequisites
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and nodejs.
+- You can check if Git is installed by running the following command in the terminal.
+- `$ git --version`
+- Clone the repo from here.
+- `$ git clone https://github.com/maenkhraisha/hotel-reservation.git`
 
-* Database creation
+# Usage
```

Comment:
- [ ] Your tests are running fine (nice work), however, there are no instructions to run them.  Kindly add a "tests" section on your readme and add the command to run your tests: ```rake rswag```:

![image](https://user-images.githubusercontent.com/57421823/201310269-ea4aa5f1-2b38-4d8c-b16b-aa20c96ffe81.png)


That way, your users will be able to run your tests 💪